### PR TITLE
cpu: add recipe

### LINKED ISSUE #29: Please include cpu in meta-rte release
**The problem you're addressing (if any)**

Using custom software/scripts on RTE is problematic:
- I need to send it to RTE first
- RTE must have correct versions of libraries etc

**Describe the solution you'd like**

- easily run my own software scripts from my PC directly on RTE
- please include [cpu](https://github.com/u-root/cpu)

**Where is the value to a user, and who might that user be?**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: recipes-go/cpu/cpu.bb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "cpu command in Go, inspired by the Plan 9 cpu command"
+
```
Comment (tomzy-0) [inline recipes-go/cpu/cpu.bb:3]: @PLangowski please add a HOMEPAGE of used project https://docs.yoctoproject.org/singleindex.html#term-HOMEPAGE

File: recipes-go/cpu/cpu.bb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+LICENSE = "BSD-3-Clause" 
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
```
Comment (tomzy-0) [inline recipes-go/cpu/cpu.bb]: @PLangowski please wrap it to 80 signs. Something like below will work.

```
LIC_FILES_CHKSUM = " \
    file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9 \
"
```

**Review by tomzy-0 — CHANGES_REQUESTED** [review]
I left review

File: recipes-go/cpu/cpu.bb (hunk 1 above)
Comment (PLangowski) [reply recipes-go/cpu/cpu.bb:3]: Done f64259079b3fe8c4fba7c5b611fd0f26c7167182

File: recipes-go/cpu/cpu.bb (hunk 2 above)
Comment (PLangowski) [reply recipes-go/cpu/cpu.bb]: done f64259079b3fe8c4fba7c5b611fd0f26c7167182

Comment (tomzy-0) [reply recipes-go/cpu/cpu.bb]: @PLangowski please pay attention to details and use the same form as provided

Comment (PLangowski) [reply recipes-go/cpu/cpu.bb]: I didn't add tab because if I do, it still exceeds 80 chars. But I guess when It's an URI it doesn't matter that much?

Comment (PLangowski) [reply recipes-go/cpu/cpu.bb]: Now it's the same as the one you provided 1cf69eacada404c45e4c17e133e7309f4448fad4

**Review by tomzy-0 — APPROVED** [review]

**Pull request merged**